docs/support-matrix.md

```diff
@@ -49,3 +49,5 @@ After you publish your plugin, open a PR that adds your plugin to the appropriat
 [10]: https://kubernetes.slack.com/messages/ark-dr
 [11]: https://github.com/heptio/ark/issues
 [12]: https://github.com/aws/aws-sdk-go/aws
+[13]: https://slack.portworx.com
```

Comment:
Is there a specific channel that would be best?

Reply:
There is no specific channel for ark, but we have a channel for kubernetes. I can link that there.